menuinst/utils.py

```diff
@@ -352,6 +350,7 @@ def wrapper_elevate(
                         f"{func.__name__}("
                         f"*{args!r},"
                         f"base_prefix={base_prefix!r},"
+                        f"_mode='system',"
```

Comment:
@mrclary - this was the bug 😬

Reply:
... when `.noadmin` is absent _and_ `user_is_admin()` is False. So when a non-admin user runs `menuinst` from the commandline? When `constructor` runs for all users, `user_is_admin()` should return True, correct?